clients/src/main/java/org/apache/kafka/common/protocol/ApiVersion.java

```diff
@@ -14,10 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.clients;
+package org.apache.kafka.common.protocol;
```

Comment:
Why move this class?

Reply:
I want to make `versionsInCommon` to be usable for common/message/ApiVersionsResponse

Reply:
Ok. I don't have a strong argument to keep it where it is today I guess.